tests/mmm/test_lift_test.py

```diff
@@ -451,7 +460,7 @@ def test_index_variable(var_dims, var_data, indices, expected) -> None:
     result = index_variable(var_dims=var_dims, var=var, indices=indices)
 
     np.testing.assert_allclose(
-        result.eval(),
+        fast_eval(result),
```

Comment:
What is the new "fast eval" ? 😎 ?

Reply:
Just this imported from Pymc

```python
def fast_eval(var):
    return function([], var, mode="FAST_COMPILE")()
```

Reply:
wow! I did not know this exit 🙌 . Thanks!